Description: make -drc packages side-installable to system-ones
 Don't overwrite official wpa_supplicant, hostapd, wpa_cli and
 wpa_passphrase when installing this packages, so append -drc
 suffix to all of them.
 .
 wpa-nano (2.1~git20131228.d8f114-4nano) unstable; urgency=low
 .
   * fixup hostapd config
Author: Christopher Roy Bratusek <nano@tuxfamily.org>
---

--- wpa-nano-2.1~git20131228.d8f114.orig/hostapd/Makefile
+++ wpa-nano-2.1~git20131228.d8f114/hostapd/Makefile
@@ -850,7 +850,7 @@ ifdef CONFIG_TENDONIN
 CFLAGS += -DCONFIG_TENDONIN
 endif

-ALL=hostapd hostapd_cli
+ALL=hostapd-drc hostapd_cli-drc

 all: verify_config $(ALL)

@@ -886,15 +886,15 @@ install: all

 BCHECK=../src/drivers/build.hostapd

-hostapd: $(BCHECK) $(OBJS)
-	$(Q)$(CC) $(LDFLAGS) -o hostapd $(OBJS) $(LIBS)
+hostapd-drc: $(BCHECK) $(OBJS)
+	$(Q)$(CC) $(LDFLAGS) -o hostapd-drc $(OBJS) $(LIBS)
 	@$(E) "  LD " $@

 ifdef CONFIG_WPA_TRACE
 OBJS_c += ../src/utils/trace.o
 endif
-hostapd_cli: $(OBJS_c)
-	$(Q)$(CC) $(LDFLAGS) -o hostapd_cli $(OBJS_c) $(LIBS_c)
+hostapd_cli-drc: $(OBJS_c)
+	$(Q)$(CC) $(LDFLAGS) -o hostapd_cli-drc $(OBJS_c) $(LIBS_c)
 	@$(E) "  LD " $@

 NOBJS = nt_password_hash.o ../src/crypto/ms_funcs.o $(SHA1OBJS) ../src/crypto/md5.o
--- wpa-nano-2.1~git20131228.d8f114.orig/wpa_supplicant/Makefile
+++ wpa-nano-2.1~git20131228.d8f114/wpa_supplicant/Makefile
@@ -15,10 +15,10 @@ CFLAGS += -I../src/utils

 -include .config

-BINALL=wpa_supplicant wpa_cli
+BINALL=wpa_supplicant-drc wpa_cli-drc

 ifndef CONFIG_NO_WPA_PASSPHRASE
-BINALL += wpa_passphrase
+BINALL += wpa_passphrase-drc
 endif

 ALL = $(BINALL)
@@ -1547,8 +1547,8 @@ wpa_priv: $(BCHECK) $(OBJS_priv)

 $(OBJS_c) $(OBJS_t) $(OBJS_t2) $(OBJS) $(BCHECK) $(EXTRA_progs): .config

-wpa_supplicant: $(BCHECK) $(OBJS) $(EXTRA_progs)
-	$(Q)$(LDO) $(LDFLAGS) -o wpa_supplicant $(OBJS) $(LIBS) $(EXTRALIBS)
+wpa_supplicant-drc: $(BCHECK) $(OBJS) $(EXTRA_progs)
+	$(Q)$(LDO) $(LDFLAGS) -o wpa_supplicant-drc $(OBJS) $(LIBS) $(EXTRALIBS)
 	@$(E) "  LD " $@

 eapol_test: $(OBJS_t)
@@ -1559,12 +1559,12 @@ preauth_test: $(OBJS_t2)
 	$(Q)$(LDO) $(LDFLAGS) -o preauth_test $(OBJS_t2) $(LIBS)
 	@$(E) "  LD " $@

-wpa_passphrase: $(OBJS_p)
-	$(Q)$(LDO) $(LDFLAGS) -o wpa_passphrase $(OBJS_p) $(LIBS_p)
+wpa_passphrase-drc: $(OBJS_p)
+	$(Q)$(LDO) $(LDFLAGS) -o wpa_passphrase-drc $(OBJS_p) $(LIBS_p)
 	@$(E) "  LD " $@

-wpa_cli: $(OBJS_c)
-	$(Q)$(LDO) $(LDFLAGS) -o wpa_cli $(OBJS_c) $(LIBS_c)
+wpa_cli-drc: $(OBJS_c)
+	$(Q)$(LDO) $(LDFLAGS) -o wpa_cli-drc $(OBJS_c) $(LIBS_c)
 	@$(E) "  LD " $@

 link_test: $(OBJS) $(OBJS_h) tests/link_test.o
